appcenter-link-scripts/src/GetReactNativeProjectConfig.js

```diff
@@ -0,0 +1,11 @@
+
+
+require('../../react-native/setupBabel')();
+
+const reactNativeCliConfig = require('../../react-native/local-cli/core/index');
+
+const GetReactNativeProjectConfig = function () {
+    return reactNativeCliConfig.getProjectConfig();
```

Comment:
This line always returns an empty object.

Reply:
Hi Murat.

To give you more background information on what happens:
The react native cli uses a config object with functions for different configuration values to control much of the cli behavior. One of those functions `getProjectConfig()` contains paths to where the projects are located.

The config object uses a default from within the react-native package. It then loads a config using the below logic and allows any of the functions defined in the loaded config to override the default. The loaded config is found using the following logic:

1) If a --config path/to/config.js is provided it loads the config from the provided path.
e.g. `react-native link --config path/to/custom/react-native-cli-config.js`
**NOTE:** This option does not currently work for the prelink / postlink scripts, as the link command does not pass the config parameter through when calling these scripts.

2) If no --config option is specified it traverses up the directory tree for the nearest ancestor directory looking for a file named `rn-cli.config.js`. If it finds one then it loads that file allowing it to override config logic.

Overriding the values in a `rn-cli.config.js` in the project root or in a config file specified via the --config flag will let the user run the react native cli commands, including the link command, when they have a file structure other than the default.